src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -409,12 +458,7 @@ public void recordActual(String actualOutcome, Success result) {
             // write out the actual outcome
             out.write(START_CELL + actualOutcome + imageLink + END_CELL);
             out.write(START_CELL + dTime + "ms / " + tTime + "ms</td>\n");
-            // write out the pass or fail result
-            if (result == Success.PASS) {
-                out.write("    <td class='pass'>Pass</td>\n");
-            } else {
-                out.write("    <td class='fail'>Fail</td>\n");
-            }
+            out.write("    <td class='" + success.toString().toLowerCase() + "'>" + success + END_CELL);
```

Comment:
This comment will cause hundreds of lines of re-write, but recordStuff shouldn't include html formatting. You could create a writer that extends whatever out is here that adds the formatting, but the generic "Here's some expected and results" method shouldn't be an HTML generator.

Reply:
Yeah, per our earlier conversation, I'll open an issue to track this